src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -2,581 +2,428 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.IO;
 using System.Diagnostics;
-
+using System.Runtime.InteropServices;
 using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Win32.SafeHandles;
 using Xunit;
 using System.Text;
 
 namespace System.Diagnostics.ProcessTests
 {
-    public partial class ProcessTest
+    public partial class ProcessTest : IDisposable
     {
-        static ProcessTest()
+
+        private const string _processName = "ProcessTest_ConsoleApp.exe";
+        private Process process;
```

Comment:
Nit: stylewise, as a private instance field, this should be ```_process```.  It's possible that when I commented on this previously in the other PR, the underscore got trimmed off in the rendering; apologies if I mislead you.